lib/onebox/templates/discourse_chat.mustache

```diff
@@ -0,0 +1,64 @@
+{{^cooked}}
```

Comment:
Is `^` a typo or new syntax? Haven't seen it before.

Reply:
Is the inverted block syntax in mustache 🙃

Reply:
Ohhh... I see, mustache! Ignore me.

Reply:
It's ok, nobody expects mustache 😄 a v-low pri todo is to replace all onebox templates with hbs